src/CarbonAware.WebApi/src/Models/EmissionsDataDTO.cs

```diff
@@ -18,8 +18,12 @@ public record EmissionsDataDTO
     [JsonPropertyName("value")]
     public double Value { get; set; }
 
-    public static EmissionsDataDTO FromEmissionsData(EmissionsData emissionsData)
+    public static EmissionsDataDTO? FromEmissionsData(EmissionsData emissionsData)
     {
+        if (emissionsData == null)
```

Comment:
Is this possible since the param isn't set to nullable? Like wouldn't a null EmissionsData cause an error when creating the object or parsing the JSON be default?

Reply:
EmissionsData is nullable right now, so we need to check when that is the case and return it back rather than trying to access properties.